Jellyfin.Api/Controllers/BrandingController.cs

```diff
@@ -0,0 +1,46 @@
+using Jellyfin.Api.Models.Branding;
+using MediaBrowser.Common.Configuration;
+using MediaBrowser.Controller.Configuration;
+using Microsoft.AspNetCore.Mvc;
+
+namespace Jellyfin.Api.Controllers
+{
+    /// <summary>
+    /// The branding controller.
+    /// </summary>
+    public class BrandingController : BaseJellyfinApiController
+    {
+        private IServerConfigurationManager _serverConfigurationManager;
+
+        /// <summary>
+        /// Initializes a new instance of the <see cref="BrandingController"/> class.
+        /// </summary>
+        /// <param name="serverConfigurationManager">Instance of ServerConfigurationManager.</param>
+        public BrandingController(IServerConfigurationManager serverConfigurationManager)
+        {
+            this._serverConfigurationManager = serverConfigurationManager;
```

Comment:
```suggestion
            _serverConfigurationManager = serverConfigurationManager;
```
Don't use `this` unless needed